### ISSUE TITLE
Moving to Alfred Gallery

### ISSUE THREAD

**Issue by zhongjis** [blacs30/bitwarden-alfred-workflow#156]
Seems like this workflow has not been updated/maintained for a while. Is there any current plan to add it to Alfred Gallery?
My knowledge on Alfred workflow is limited. But I am more than willing to learn and help with the process/maintainence

**Comment by blacs30** [comment]
Thanka for this recommendation. Honestly, I haven't heard about the gallery until now. I am still using the workflow daily. Of course it would be great if it could contain more features... Unfortunately I don't have much time for new features currently for this project.

**Comment by zhongjis** [comment]
![Screenshot 2023-05-30 at 13 34 05](https://github.com/blacs30/bitwarden-alfred-workflow/assets/23202735/9d334cf2-bf7b-4a92-b7d5-a2c9ecde8fdd)

https://alfred.app/submit/

just some quick links for your reference. Basically you get a fancy star with it and have alfred manages brew dependencies etc.
![Screenshot 2023-05-30 at 13 36 52](https://github.com/blacs30/bitwarden-alfred-workflow/assets/23202735/13262851-84d4-43d5-a05b-276ca0186d1d)

Let me know if you are interested. if yes I can suggest your workflow on Alfred forum and see what they say.

**Comment by zhongjis** [comment]
kk ill starting analyzing the repo according to their requirements and will post the forum link here once created

**Comment by zhongjis** [comment]
quick update the plugin is currently broken on my setup. might need to take a bit setting things up again...

**Comment by zhongjis** [comment]
https://www.alfredforum.com/topic/20531-bitwarden-alfred-workflow/

**Comment by zhongjis** [comment]
seems like dev locally section no longer relevant as go no longer support `go get` out of module.
I got it setup with `go install` however im getting the below error when running any configuration (`.bwauth`/`.bwconfig`). any idea?

```bash
[12:32:56.185] ERROR: Bitwarden v2[Script Filter] Code 127: /Users/test/Library/Caches/com.runningwithcrayons.Alfred/Workflow Scripts/8487D041-2B70-4985-9D97-A9FA845A1E41: line 1: ./bitwarden-alfred-workflow: No such file or directory
```

**Comment by blacs30** [comment]
That's the part where both binaries are compiled and than merged:

https://github.com/blacs30/bitwarden-alfred-workflow/blob/master/Makefile#L25-L30

**Comment by blacs30** [comment]
I get a 404 for the link to the project board.

**Comment by blacs30** [comment]
> https://www.alfredforum.com/topic/20531-bitwarden-alfred-workflow/

seems this will require some work. so auto-update is discouraged as part of the workflow? is it included in Alfred now and Alfred will check for updates of workflows based on the gallery and the workflow versions there?

**Comment by zhongjis** [comment]
> I get a 404 for the link to the project board.

can you try again. I just made it public

**Comment by zhongjis** [comment]
> > https://www.alfredforum.com/topic/20531-bitwarden-alfred-workflow/
> 
> seems this will require some work. so auto-update is discouraged as part of the workflow? is it included in Alfred now and Alfred will check for updates of workflows based on the gallery and the workflow versions there?

yes. this would also take some time. I am planning to migrate the configuration first to get familiar with everything. seems like its simple plist change for me now

**Comment by zhongjis** [comment]
> That's the part where both binaries are compiled and than merged:
> 
> https://github.com/blacs30/bitwarden-alfred-workflow/blob/master/Makefile#L25-L30

got it. im not familiar with make files but `make universal-binary` do the trick
